SPEC/FILES.md

```diff
@@ -1,141 +1,41 @@
 # Files API
 
-> The files API enables users to use the File System abstraction of IPFS.
-
+> The files API enables users to use the File System abstraction of IPFS. There are two Files API, one at the top level, the original `add`, `cat`, `get` and `ls`, and another behind the [`files`, also known as MFS](https://github.com/ipfs/specs/issues/98). [We are currently going through a revamping process of this APIs to make them more user friendly](https://github.com/ipfs/interface-ipfs-core/issues/284).
+
+- [add](#add)
+- [addReadableStream](#addreadablestream)
+- [addPullStream](#addpullstream)
+- [addFromFs](#addFromFs)
+- [addFromUrl](#addFromFs)
```

Comment:
Added the addFromFs (no one should have to do this: https://github.com/ipfs/js-ipfs/blob/master/src/cli/commands/files/add.js#L18-L102) and addFromUrl. 

Once ipfsx is ready, we can squash all these 5 adds into one function.